tests/relay_integration/test_integration.py

```diff
@@ -216,3 +220,17 @@ def test_transaction(self):
                     "total.time": {"unit": "millisecond", "value": pytest.approx(1050, abs=2)},
                 }
             }
+
+    def test_project_config_compression(self):
+        # Populate redis cache with compressed config:
+        with override_options({redis.COMPRESSION_OPTION: 1.0}):
+            invalidate_project_config(public_key=self.projectkey, trigger="test")
+
+        # Disable project config endpoint, to make sure Relay gets its data
+        # from redis:
+        with mock.patch(
+            "sentry.api.endpoints.relay.project_configs.RelayProjectConfigsEndpoint.post"
+        ):
+            event_data = {"message": "hello", "timestamp": iso_format(before_now(seconds=1))}
+            event = self.post_and_retrieve_event(event_data)
+            assert event.message == "hello"
```

Comment:
This test fails because apparently, the Relay instance connects to a different [redis db](https://redis.io/commands/select/) than the one that sentry writes to.